components/ResultsPage/WillBeEligible.tsx

```diff
@@ -19,6 +19,28 @@ export const WillBeEligible: React.VFC<{
   const multipleOAS_GIS =
     futureResults.filter((obj) => !!obj[Object.keys(obj)[0]]['oas']).length > 1
 
+  for (let i = futureResults.length - 1; i >= 0; i--) {
+    if (i > 0) {
+      if (
+        Object.values(Object.values(Object.values(futureResults)[i])[0])
+          .length != 1
+      ) {
+        if (
+          Object.values(Object.values(futureResults[i])[0])[0].entitlement
+            .result ==
+            Object.values(Object.values(futureResults[i - 1])[0])[0].entitlement
+              .result &&
+          Object.values(Object.values(futureResults[i])[0])[1].entitlement
+            .result ==
+            Object.values(Object.values(futureResults[i - 1])[0])[1].entitlement
+              .result
+        ) {
+          futureResults.pop()
+        }
+      }
+    }
+  }
+
```

Comment:
look into this code to replace the one above 

      // remove duplicates from array
```
      collapsedDetails = collapsedDetails.filter(
        (value, index, self) =>
          index === self.findIndex((t) => t.heading === value.heading)
      )
```

Reply:
maybe the code above wont work, nevermind